releases.yaml

```diff
@@ -3,6 +3,7 @@ latest:
   name: "Mantic Minotaur"
   short_version: "23.10"
   full_version: "23.10"
+  desktop_version: "23.10.1"
```

Comment:
If this is a point release we should just update the full version

Reply:
Lukasz said this version upgrade applies only to Desktop image. All other images remain 23.10.